hedyorg/hedy
# 💻 Allow HTML mode in wysiwyg editor

Fixes #5491 
Fixes #5437  

**How to test?**
- customize an adventure and add some html
- preview that adventure from the `for-teacher` page to see how it's rendered.
